src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto" , sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  max-width: 100%;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.menu-moyo-list {
+  display: flex;
+}
+
+.nav-moyo-item {
+  margin-left: 20px;
+  list-style: none;
+  display: flex;
+}
+
+.nav-moyo-item:first-child {
+  margin-left: 0;
+}
+
+.nav-moyo-link {
+  display: flex;
+  align-items: center;
+  padding: 23px 0;
+  cursor: pointer;
+  text-transform: uppercase;
+  text-decoration: none;
+  line-height: 14px;
+  color: #000;
+  transition: x 0.5s;
```

use only those properties that you really change
```suggestion
  transition: color 0.5s;
```

---

all ready